src/odemis/acq/stitching/_registrar.py

```diff
@@ -282,7 +282,7 @@ def _estimateROI(self, shift):
     def _estimateMatch(self, imageA, imageB, shift):
         """
         Returns an estimation of the similarity between the given images
-        when the second is shifted by the shift value. It is used to assess 
+        when the second is shifted by the shift value. It is used to assess
         the quality of a shift measurement by giving the shifted image.
```

Comment:
I don't understand how the quality of the shift measurement is assessed by "giving the shifted image"?